packages/select/README.md

```diff
@@ -0,0 +1,128 @@
+
+| Prop           | Type                                               | Description                                                                                                   | Default    |
+| -------------- | -------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- | ---------- |
+| `children`     | `node`                                             | `<Option />` and `<OptionGroup />` elements.                                                                  |            |
+| `className`    | `string`                                           | Adds a className to the outermost element.                                                                    |            |
+| `darkMode`     | `boolean`                                          | Determines whether or not the component will appear in dark mode.                                             | `false`    |
+| `size`         | `'xsmall'` \| `'small'` \| `'normal'` \| `'large'` | Sets the size of the component's elements.                                                                    | `'normal'` |
+| `id`           | `string`                                           | id associated with the Select component.                                                                      |            |
+| `label`        | `string`                                           | Text shown in bold above the input element.                                                                   |            |
+| `description`  | `string`                                           | Text that gives more detail about the requirements for the input.                                             |            |
+| `placeholder`  | `string`                                           | The placeholder text shown in the input element when an option is not selected.                               | `'Select'` |
+| `disabled`     | `boolean`                                          | Disables the component from being edited.                                                                     | `false`    |
+| `value`        | `string`                                           | Sets the `<Option />` that will appear selected and makes the component a controlled component.               | `''`       |
+| `defaultValue` | `string`                                           | Sets the `<Option />` that will appear selected on page load when the component is uncontrolled.              | `''`       |
+| `onChange`     | `function`                                         | A function that gets called when the selected value changes. Receives the value string as the first argument. | `() => {}` |
+| `readOnly`     | `boolean`                                          | Disables the console warning when the component is controlled and no `onChange` prop is provided.             | `false`    |
+
+## Option
```

Comment:
Can this be a single `#` instead? Trying to standardize readmes for parsing for website directory